SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -226,16 +226,16 @@ bool FLocalDeploymentManager::FinishLocalDeployment(FString LaunchConfig, FStrin
 {
 	// runtime.exe --config=squid_config.json --snapshot=snapshots\default.snapshot --worker-port 8018 --http-port 5006 --grpc-port 7777
 
-	LaunchConfig = TEXT("B:\\UnrealEngine425\\Samples\\UnrealGDKExampleProject\\spatial\\squid_config.json");
-	FString SnapshotPath = TEXT("B:\\UnrealEngine425\\Samples\\UnrealGDKExampleProject\\spatial\\snapshots\\default.snapshot");
+	LaunchConfig = TEXT("C:\\Dev\\UnrealEngine\\Engine\\Plugins\\UnrealGDK\\SpatialGDK\\Binaries\\ThirdParty\\Improbable\\Programs\\squid_config.json");
```

Comment:
will this path be changed later? this is currently hardcoded to what I assume is the path on your local machine?

Reply:
Yep, it will be changed! Some functionality is not yet in this branch/not done